lib/functions.sh

```diff
@@ -43,12 +43,19 @@ read_stdin_command_and_verify_signature() {
 
     # this will read from standard input of the service, the data should be
     # considered untrusted
-    tr -d '\r' | awk -b \
-            -v in_command=0 \
+    LC_ALL=C head -c 65535 |
+    LC_ALL=C tr '\r\0' '\001\001' |
```

Comment:
I'm pretty sure this will break all the calls. Removing `\r` was there for a reason, now chocking on it is wrong thing to do.

Reply:
Sorry about that, will fix once I have gotten some sleep.

Reply:
Is there a reason this cannot be done in the sending qube?

Reply:
Fixed by having the webhook remove `\r`.